code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -35,6 +35,73 @@ emp_act
 
 	return (..(P , def_zone))
 
+/mob/living/carbon/human/screwdriver_act(mob/user, obj/item/I)
+	if(!can_operate(src))  //Checks if mob is lying down on table for surgery
+		return
+	if(!ismachine(src))

```

Comment:
I think this would inhibit the "makeshift surgery" use of screwdrivers in lieu of bone gel

Reply:
Or am I misremembering, do surgery steps go through a different call stack

Reply:
Tested this and screwdrivers as bone gel still work